ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1448,8 +1441,9 @@ class JITModBuilder {
     // the trace.
     assert(!isOutlining());
 
-    Builder.CreateRet(ConstantInt::get(Type::getInt8Ty(JITMod->getContext()),
-                                       TRACE_RETURN_SUCCESS));
+    // If the trace succeeded return a nullpointer instead of a reconstructed
```

Comment:
nitpicking: `nullpointer` -> `null pointer`. Sorry :P

Reply:
Fixed in [c45e5e6](https://github.com/ykjit/yk/pull/578/commits/c45e5e644b2bc6b43c9d456a8192a5cc3ddbaff8)